tests/components/vacuum/common.py

```diff
@@ -41,20 +42,47 @@ def locate(hass, entity_id=None):
     hass.services.call(DOMAIN, SERVICE_LOCATE, data)
 
 
+@callback
+@bind_hass
+def async_locate(hass, entity_id=None) -> None:
```

Comment:
Please don't add things to this file. It has been put here when we moved it out of core, but we shouldn't maintain nor extend it. One day we'll rewrite tests and remove this file 🤣